examples/jsm/loaders/LDrawLoader.js

```diff
@@ -899,6 +899,12 @@ class LDrawLoader extends Loader {
 
 	parse( text, path, onLoad ) {
 
+		if ( ! this.fileMap ) {
+
+			this.fileMap = {};
+
+		}
+
```

Comment:
Why don't we initialize `fileMap` to `{}` rather than `null` in the constructor. Then we can remove the conditional initialization in the `load` function, as well.

Reply:
I don't know the reason. I'll update it.